WindowsAppRuntime.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio Version 17
-VisualStudioVersion = 17.0.31919.166
+# Visual Studio Version 16
```

Comment:
why do we need to change the visual studio version?

Reply:
I don't think we do this in purpose. I bet this depends on the version the latest person to commit to this PR is using.